test/JDBC/input/functions/string_functions/reverse-vu-verify.sql

```diff
@@ -0,0 +1,258 @@
+-- NULL
+SELECT reverse(NULL)
```

Comment:
Let's add SELECT reverse('') test as well. Also, please add tests like:
SELECT reverse(concat('a','b')) -- something like this.

Reply:
Done